core/controllers/editor_test.py

```diff
@@ -553,9 +553,13 @@ def test_exploration_download_handler_for_default_exploration(self):
                 'The title for ZIP download handler test!.yaml').read())
 
         # Check download to JSON.
-        exploration.update_objective('Test JSON download')
-        exp_services._save_exploration(  # pylint: disable=protected-access
-            owner_id, exploration, '', [])
+        exp_services.update_exploration(
+            owner_id, exp_id, [
+                exp_domain.ExplorationChange({
+                    'cmd': exp_domain.CMD_EDIT_EXPLORATION_PROPERTY,
+                    'property_name': 'objective',
+                    'new_value': 'Test JSON download',
+                })], '')
```

Comment:
(I missed this one) Can you please add a meaningful commit message here? (Maybe "Updates exploration objective")

Reply:
Will do that